src/ScriptHandler.php

```diff
@@ -9,7 +11,24 @@ class ScriptHandler
     public static function postUpdate(Event $event)
     {
         $composer = $event->getComposer();
-        $mapper = new Mapper(getcwd(), getcwd().'/www');
+        $extra = $composer->getPackage()->getExtra();
+        $config = isset($extra['tangler']) ? $extra['tangler'] : array();
+        $project = isset($config['project']) ? $config['project'] : getcwd();
+        $drupal = isset($config['drupal']) ? $config['drupal'] : getcwd() . DIRECTORY_SEPARATOR . 'www';
+        $copy = isset($config['copy']);
+        if (!$copy) {
+            // Check for symlink ability. If we don't have it, hard copy.
+            // This is great for VMs on Windows hosts.
+            $fs = new Filesystem();
+            try {
+                $fs->symlink($drupal, 'test');
+                $fs->remove('test');
+            }
+            catch (IOException $e) {
+                $copy = TRUE;
+            }
+        }
+        $mapper = new Mapper($project, $drupal, $copy);
```

Comment:
Mapper is part of the same namespace, so I can call it without a use statement. Checkout https://github.com/generalredneck/drupal-tangler/blob/windows/src/Mapper.php#L3. WIth that said, Composer has autoloading capabilities built in using Symphony's autoloader, and so when I call Mapper it includes the right file.